### PR TITLE
Module1-portfolio skeleton

### PR DESCRIPTION
**Pull request by mwakiogeofrey**
Added css,html,and images file
Added linters to project
Added some information about myself

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                <li><i class="fa-brands fa-angellist"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-medium"></i></li>
+            </ul>
+            <img class="group" src="images/Group.png" alt="">   
+        </div>
+    </div>
+  
+    
+</body>
```
Comment (ShadyShawkat) [inline index.html:36]: -  [ ] Kindly make sure to use the same font style and font family used in the design.
-  [ ] Please make sure that the list of social media icons appears on the display. As you can see I can't see them.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <ul class="icon">
+                <li><i class="fa-brands fa-github"></i></li>
+                <li><i class="fa-brands fa-linkedin-in"></i></li>
+                <li><i class="fa-brands fa-angellist"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-medium"></i></li>
+            </ul>
+            <img class="group" src="images/Group.png" alt="">   
+        </div>
+    </div>
```
Comment (ShadyShawkat) [inline index.html:33]: - [ ] Please use a `section` tag for section instead of a `div`.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @mwakiogeofrey, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                <li><i class="fa-brands fa-github"></i></li>
+                <li><i class="fa-brands fa-linkedin-in"></i></li>
+                <li><i class="fa-brands fa-angellist"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-medium"></i></li>
+            </ul>
+        </div>
+            <img class="group" src="images/Group.png" alt="">  
+        </section>
+    </nav>
```
Comment (levy002) [inline index.html:36]: - [ ] Kindly do not wrap every website content into the `nav` tag. The previous reviewer suggested wrapping the websites' navigation links only into the `nav` tag which is semantically correct. Kindly read [this lesson ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)for more info.

```suggestion
        <header class="box1">
              <a class="logo" href="">Mwakio</a>
            <nav>
               <img class="enabled" src="images/Enabled.png" alt="">
            </nav>    
        </header>
        <section class="box2">
            <h1 class="info">Hey There.
                I’m Mwakio.</h1>
            <h2 class="intro">I'm a Software Developer</h2>
            <p class="information">I can help you build a product , feature <br> or website Look through some of my <br> work and experience! If you  like what you<br> see and have a project you need coded,<br> 
                don’t hesitate to contact me.</p>
          <div>
            <ul class="icon">
                <li><i class="fa-brands fa-github"></i></li>
                <li><i class="fa-brands fa-linkedin-in"></i></li>
                <li><i class="fa-brands fa-angellist"></i></li>
                <li><i class="fa-brands fa-twitter"></i></li>
                <li><i class="fa-brands fa-medium"></i></li>
            </ul>
        </div>
            <img class="group" src="images/Group.png" alt="">  
        </section>
    </nav>
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          <div>
+            <ul class="icon">
+                <li><i class="fa-brands fa-github"></i></li>
+                <li><i class="fa-brands fa-linkedin-in"></i></li>
+                <li><i class="fa-brands fa-angellist"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-medium"></i></li>
+            </ul>
+        </div>
+            <img class="group" src="images/Group.png" alt="">
```
Comment (levy002) [inline index.html:34]: - [OPTIONAL] Kindly make sure the website background image is inserted on the page using CSS Background properties. Kindly read[ this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) for more info.

**Review by franclobo — COMMENTED** [review]
Hi @mwakiogeofrey, 

terminator

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional README file, you should complete each section with information about the current project and remove the guidelines not just copy and paste the template. All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo. It includes the live demo section unless you have deployed your app you should replace `(Live demo)[URL]` with `Coming soon...`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**